app/routes/_gcn.circulars.moderation.$circularId.$requestor.tsx

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import type { ActionFunctionArgs, LoaderFunctionArgs } from '@remix-run/node'
+import { Form, redirect, useLoaderData } from '@remix-run/react'
+import { Button, ButtonGroup } from '@trussworks/react-uswds'
+import { DiffMethod } from 'react-diff-viewer-continued'
+
+import { getUser } from './_gcn._auth/user.server'
+import type {
+  Circular,
+  CircularChangeRequest,
+} from './_gcn.circulars/circulars.lib'
+import {
+  approveChangeRequest,
+  deleteChangeRequest,
+  get,
+  getChangeRequest,
+  moderatorGroup,
+} from './_gcn.circulars/circulars.server'
+import { getFormDataString } from '~/lib/utils'
+
+const DiffViewer = require('react-diff-viewer-continued').default
```

Comment:
```suggestion
import DiffViewer from 'react-diff-viewer-continued'
```

Reply:
This is intentional. When the `DiffViewer` is imported, it throws this error:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/77542b98-52bf-4cdc-8363-f4c155b8f71a)

I can look into making a PR in their repo to fix this, but for now this is the solution

Reply:
Yuck. Are there any other React diff viewers out there we could choose from?

Reply:
@Courey, you had poked around at some packages that we had difficulty using with ESM-style imports. Does this error message suggest anything to you?

Reply:
This seems to be the main one, there was a prior iteration that this was built off of. Corey and I went down this rabbit hole a while ago, though I don't remember where we landed with this other than this style of import

Reply:
There is an open issue about this already: https://github.com/Aeolun/react-diff-viewer-continued/issues/40

Reply:
There's yet another fork here that might have proper ESM support: https://www.npmjs.com/package/react-diff-viewer-esm

To be honest I am a bit worried about this dependency given the large number of forks. Also the code is quite dated (it's not even written as a React functional component, for example).

Reply:
What do you suggest? I think its definitely useful to show the changes, but not married to the idea of using this package specifically. The esm one might be good, but I don't see a link on their page to their code

Reply:
Played around a bit, and it looks like under the hood that package is just doing some fancy stuff around https://www.npmjs.com/package/diff . Here is what I got by just using uswds colors and the jsdiff package:

![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/cee2965a-e860-4332-802c-951f1bfee998)

Reply:
That looks pretty nice too.